content/reference/endpoints/rentals.md

```diff
@@ -127,84 +186,132 @@ zip                             | String   | Read       | Rental's zip code.
+nightly_rate_map                | Integer  | Read       | Nightly Rate Map's ID related to the Rental |
+mid_term_rate_map               | Integer  | Read       | Mid Term Rate Map's ID related to the Rental |
+photos                          | Array    | Read       | Photos' IDs related to the Rental |
+reviews                         | Array    | Read       | Reviews' IDs related to the Rental |
+host_reviews                    | Array    | Read       | Host Reviews' IDs related to the Rental |
+special_offers                  | Array    | Read       | Special Offers' IDs related to the Rental |
+rates                           | Array    | Read       | Rates' IDs related to the Rental |
+rentals_amenities               | Array    | Read       | Rentals Amenities' IDs related to the Rental |
+rentals_fees                    | Array    | Read       | Rentals Fees' IDs related to the Rental |
+bathrooms                       | Array    | Read       | Bathrooms' IDs related to the Rental |
+bedrooms                        | Array    | Read       | Bedrooms' IDs related to the Rental |
+living_rooms                    | Array    | Read       | Living Rooms' IDs related to the Rental |
+rentals_tags                    | Array    | Read       | Rentals Tags' IDs related to the Rental |
+rental_contacts                 | Array    | Read       | Rental Contacts' IDs related to the Rental |
+videos                          | Array    | Read       | Videos' IDs related to the Rental |
+--------------------------------|----------|------------|-------------|
+id                              | Integer  | Read       | Rental's ID. |
+amenity_ids                     | Array    | Write      | Amenity ids related to the rental. |
+destination_id                  | Integer  | Write      | Destination id related to the rental. |
+payment_gateway_id              | Integer  | Write      | PaymentGateway id related to the rental. |
+rates_table_id                  | Integer  | Write      | Rates Table id related to the rental. |
+rentals_contact_id              | Integer  | Write      | Contact id related to the rental. |
+rentals_tag_ids                 | Array    | Write      | Tag ids related to the rental. |
+absolute_min_price              | [Decimal](/reference/enums#formats) | Read/Write | Rental's minimum price based on `base_rate_kind`. | Must be greater than or equal to 0 and less than or equal to `base_rate` 
+address1                        | String   | Read/Write | Rental's primary address. |
+address2                        | String   | Read/Write | Rental's optional address. |
+base_rate                       | [Decimal](/reference/enums#formats) | Read/Write | Rental's base rate, used by the rates table to calculate the seasonability and discounted prices. We recommend using the highest price as base rate | Must be greater than 0 if present
```

Comment:
>We recommend using the highest price as base rate

why such recommendation?

Reply:
@adamgrad not sure but it was there before in the docs, it's not a new recommendation. Would it be better to remove it?

Reply:
@Azdaroth I am quite familiar with the concept of base_rate and I don't quite understand this recommendation, so I believe it's going to be more confusing for the integrating people, so I'd rather remove it.